.github/workflows/install/action.yml

```diff
@@ -1,27 +0,0 @@
-name: "Install"
-description: "Installs the inovex-elements packages"
```

Comment:
In comparison to the deleted `install` script, the Node-Binaries do not seem to be cached in the `install-dependencies` script, which results in the pipeline being approx. 30 sec slower. Could you have a look at that?

Before:
![image](https://user-images.githubusercontent.com/22963121/184900836-c7ce5d22-b4ca-4df9-8921-a6893323a4f7.png)

After:
![image](https://user-images.githubusercontent.com/22963121/184901120-6dacddd9-4f26-4c69-b998-640d1eb90626.png)

Reply:
Compared to the old `install` routine, we are now using the node version specified in `.nvmrc`.  I think GitHub caches only the latest node major versions which is `v14.20.0`. In my point of view we have following solution to fix this:

- not using the `.nvmrc` in pipline and allow any version of `v14.x`
- update version in `.nvmrc` to '14.20.0' (is for example 14.x allowed in this file?)

Reply:
Let's try to use `v14` in our `nvmrc`. According to [this comment](https://github.com/nvm-sh/nvm/issues/995#issuecomment-506818398) it should work just fine

Reply:
It seems this works, but Windows systems this feature is not supported. On Windows `nvm` wants to use `v14.0.0`.